test/matfile.jl

```diff
@@ -1,12 +1,12 @@
-# Test MATLAB MAT file I/O
+using MATLAB
+using Base.Test
 
-using MATLAB, Compat, Base.Test
-import Compat: ASCIIString
+# test MMAT file I/O
 
-a = Int32[1 2 3; 4 5 6]
+a = [1 2 3; 4 5 6]
```

Comment:
Do `Int` and `Int32` have the same representation in a mat file? AFAIK matlab also support 32 and 64bit integer types even though it's not used by default.